src/app/common/KFilterBar.vue

```diff
@@ -453,52 +454,52 @@ function areFieldsSemanticallyIdentical(fieldsA: Fields, fieldB: Fields): boolea
 
 .k-suggestion-box {
   position: absolute;
-  top: calc(100% + var(--spacing-xxs));
+  top: calc(100% + $kui-space-20);
   right: -1px;
   left: -1px;
   z-index: 1;
-  padding: var(--spacing-xxs);
-  background-color: var(--white);
-  border: 1px solid var(--grey-400);
+  padding: $kui-space-20;
+  background-color: $kui-color-background;
+  border: $kui-border-width-10 solid $kui-color-border;
   border-radius: 3px;
 }
 
 .k-filter-bar-error {
-  padding: var(--spacing-xxs) var(--spacing-xs);
-  color: var(--red-700);
+  padding: $kui-space-20 $kui-space-40;
+  color: $kui-color-text-danger;
 }
 
 .k-submit-query-button {
   align-self: stretch;
   text-align: left;
-  padding: var(--spacing-xxs) var(--spacing-xs);
+  padding: $kui-space-20 $kui-space-40;
   border-radius: 3px;
 }
 
 .k-filter-bar-error:not(:last-child),
 .k-submit-query-button:not(:last-child) {
-  margin-bottom: var(--spacing-xxs);
-  border-bottom: 1px solid var(--grey-300);
-  padding-bottom: var(--spacing-xxs);
+  margin-bottom: $kui-space-20;
+  border-bottom: $kui-border-width-10 solid $kui-color-border;
+  padding-bottom: $kui-space-20;
 }
 
 .k-suggestion-list {
   display: flex;
   flex-direction: column;
-  gap: var(--spacing-xxs);
+  gap: $kui-space-20;
 }
 
 .k-suggestion-list-item {
   position: relative;
   display: flex;
   align-items: center;
-  padding: var(--spacing-xxs) 0 var(--spacing-xxs) var(--spacing-xs);
+  padding: $kui-space-20 0 $kui-space-20 $kui-space-40;
 }
 
 .k-submit-query-button-is-selected,
 .k-suggestion-list-item-is-selected {
-  color: var(--white);
-  background-color: var(--blue-500);
+  color: $kui-color-text-inverse;
+  background-color: $kui-color-border-primary;
```

Comment:
issue: you're using a border color token for the background color

Reply:
Fixed in https://github.com/kumahq/kuma-gui/pull/1314/commits/93a38ec687bb97965d618522e6447fa1efcb0e5e.